extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfo.java

```diff
@@ -386,6 +392,11 @@ private static StreamInfo extractOptionalData(StreamInfo streamInfo, StreamExtra
     private List<StreamSegment> streamSegments = new ArrayList<>();
     private List<MetaInfo> metaInfo = new ArrayList<>();
 
+    /**
+     * Preview frames, e.g. for the storyboard / seekbar thumbnail preview
+     */
+    private List<Frameset> previewFrames = new ArrayList<>();
```

Comment:
I think these should also all be `emptyList()`, since they are never modified but only get/set.

Reply:
Why are all other fields (above) then also ``new ArrayList<>()``? 
Shouldn't these be changed to?

Reply:
Yes, they should